hoot-core/src/main/cpp/hoot/core/io/PartialOsmMapReader.h

```diff
@@ -63,6 +63,8 @@ class PartialOsmMapReader : public OsmMapReader, public ElementInputStream
    */
```

Comment:
A lot of partial readers used initializePartial, but it wasn't part of the PartialOsmMapReader interface like it is with PartialOsmMapWriter.